lib/stripe/issuing/card.ex

```diff
@@ -28,22 +28,20 @@ defmodule Stripe.Issuing.Card do
           last4: String.t(),
           livemode: boolean,
           metadata: Stripe.Types.metadata(),
-          name: String.t(),
```

Comment:
looks like name is still a valid field?

https://stripe.com/docs/api/issuing/cards/object